apps/api/src/interfaces/index.ts

```diff
@@ -77,7 +77,7 @@ export type EventPlanDocument = {
   startDate: string;
   endDate: string;
   hostId: UserId;
-  invitees: UserId[] | Email[];
+  inviteesByUserId: UserId[] | Email[];
```

Comment:
This could be confusing if we get emails and we're referring to it by userId.

Reply:
Yes, don't want to store other people's emails on firestore, so we should let the api handle the conversion. For example when creating an event-plan we give a list of emails which our api converts to a list of user ids and stores in firestore. So our docs should store user ids exclusively when pertaining to other users

Reply:
Yeah, I'm not sure why the email type is there, It was an over sight, I'll remove it